frontend/src/pages/org/dashboard.ts

```diff
@@ -69,36 +78,59 @@ export class Dashboard extends LiteElement {
           ${this.renderCard(
             msg("Storage"),
             (metrics) => html`
-              <div class="font-semibold mb-3">
-                <sl-format-bytes
-                  value=${metrics.storageUsedBytes ?? 0}
-                ></sl-format-bytes>
-                ${msg("Used")}
-              </div>
+              ${when(metrics.storageQuotaBytes, () =>
+                this.renderStorageMeter(metrics)
+              )}
               <dl>
-                ${this.renderStat({
-                  value: metrics.archivedItemCount,
-                  singleLabel: msg("Archived Item"),
-                  pluralLabel: msg("Archived Items"),
-                  iconProps: { name: "file-zip-fill" },
-                })}
+                ${when(
+                  !metrics.storageQuotaBytes,
+                  () => html`
+                    ${this.renderStat({
+                      value: html`<sl-format-bytes
+                        value=${metrics.storageUsedBytes ?? 0}
+                        display="narrow"
+                      ></sl-format-bytes>`,
+                      singleLabel: msg("of Data Stored"),
+                      pluralLabel: msg("of Data Stored"),
+                      iconProps: { name: "database" },
```

Comment:
Was planning on using the HDD icon for the storage tab of org settings in the future?  Would replicate here instead of `database`?

```suggestion
                      iconProps: { name: "device-hdd-fill" },
```